package.json

```diff
@@ -3,38 +3,37 @@
   "version": "1.8.5-SNAPSHOT",
   "description": "The Open MCT core platform",
   "devDependencies": {
-    "@braintree/sanitize-url": "^5.0.2",
-    "@percy/cli": "^1.0.0-beta.73",
-    "@percy/playwright": "^1.0.1",
-    "@playwright/test": "^1.18.0",
-    "allure-playwright": "^2.0.0-beta.14",
+    "@braintree/sanitize-url": "5.0.2",
+    "@percy/cli": "1.0.0-beta.73",
+    "@percy/playwright": "1.0.1",
+    "@playwright/test": "1.18.0",
+    "allure-playwright": "2.0.0-beta.14",
     "angular": ">=1.8.0",
     "angular-route": "1.4.14",
     "babel-eslint": "10.1.0",
-    "comma-separated-values": "^3.6.4",
-    "concurrently": "^3.6.1",
-    "copy-webpack-plugin": "^10.2.0",
-    "cross-env": "^6.0.3",
-    "css-loader": "^4.0.0",
+    "comma-separated-values": "3.6.4",
+    "copy-webpack-plugin": "10.2.0",
+    "core-js": "3.20.3",
```

Comment:
This is not a direct dependency but one which is used in many of the webpack / linting libraries